app/migrations/seed.py

```diff
@@ -265,25 +265,36 @@ def update_or_create_menu(name, description, parent_id=None):
     print('Creating Business Categories')
     print('**********************************************************************')
     business_categories = [
-        {'name': 'Food', 'icon': 'message'},
-        {'name': 'Water', 'icon': 'message'},
-        {'name': 'Energy', 'icon': 'message'},
-        {'name': 'Education', 'icon': 'message'},
-        {'name': 'Health', 'icon': 'message'},
-        {'name': 'General shop', 'icon': 'message'},
-        {'name': 'Environment', 'icon': 'message'},
-        {'name': 'Transport', 'icon': 'message'},
-        {'name': 'Labour', 'icon': 'message'},
-        {'name': 'Other', 'icon': 'message'},
+        {'name': 'Food', 'icon': 'message', 'translations': {
```

Comment:
should we do translations this way? or should we have a `translations_key` to do i18n? if nothing passed default to name

Reply:
I'm wondering if that does not make is harder for someone using the transfer usage api to add a new transfer usage as they would then also have to edit the yml files?

Reply:
the nice thing about i18n is the fallbacks, but yeah that could be annoying to use

Reply:
So should I leave it like this? @enjeyw @wangfowen

Reply:
I vote leave as is - feels like a much better UX to be able to update transfer usages without having to change YML (we change usages for the oxfam instance pretty regularly).

Reply:
and then for translations, we just manually query db to add them in?